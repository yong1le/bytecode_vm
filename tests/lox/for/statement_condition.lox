// [line 3] expect parse error: Expect expression.
// [line 3] expect parse error: Expect ';' after expression.
for (var a = 1; {}; a = a + 1) {}
