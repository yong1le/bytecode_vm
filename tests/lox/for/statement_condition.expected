[line 3]: Error at '{': Expected expression.
[line 3]: Error at ')': Expected Semicolon.
